IssueTriage.md

```diff
@@ -68,7 +68,7 @@ This is a somewhat subjective label and is interpreted in conjunction with Githu
 ## Issues ready to claim
 
 This state indicates that bugs/feature requests have reached the level of quality
-required for a contributor to begin writing code against (you can easily [filter for this list](https://github.com/meteor/meteor/labels/confirmed) by using the `confirmed` label).
+required for a contributor to begin writing code against (you can easily filter for [bugs](https://github.com/meteor/meteor/labels/confirmed) or [feature requests](https://github.com/meteor/meteor-feature-requests/labels/confirmed) that are ready to claim, by using the `confirmed` label).
```

Comment:
Oh wow! That's bizarre - and good to know! I thought I was losing it so I just upped my caffeine dose for today 🙂. Thanks!